src/Orleans/Telemetry/TelemetryLogConsumer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Net;
+
+namespace Orleans.Extensions.Logging
+{
+    /// <summary>
+    /// Forward trace log calls to the telemetry abstractions. This will be replaced by an ILoggerProvider in the future.
+    /// </summary>
+    public class TelemetryLogConsumer : ILogConsumer, IFlushableLogConsumer, ICloseableLogConsumer
+    {
```

Comment:
this class can be internal, since it is going to be removed in the future?

Reply:
I was going to say yes, but the reality is that as as standalone PR, this makes sense to be public. Sure, it will o away very soon and before we release it, which is good, as there will be no breaking change, but logically if we were to release tomorrow with this new API, then this class needs to be public

Reply:
make sense. I was on a more conservative thinking on preserving original logic